spec/lib/tasks/kids_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "kids:promote_minor_accounts", type: :task do
+  let(:organization) { create(:organization, available_authorizations: %w(dummy_authorization_handler)) }
+  let(:user) { create(:user, :admin, :confirmed, organization:) }
+  let(:other_user) { create(:user, :admin, :confirmed, organization:) }
+  let(:minor) { create(:minor, tutor: user, organization:) }
+  let(:minor_to_promote) { create(:user, :confirmed, organization:) }
+  let!(:minor_to_promote_account) { create :minor_account, tutor: user, minor: minor_to_promote }
+  let!(:minor_data_to_promote) { create :minor_data, user: minor_to_promote, birthday: }
+  let(:birthday) { 15.years.ago }
+  let(:enable_minors_participation) { false }
+  let!(:minors_organization_config) { create(:minors_organization_config, organization:, enable_minors_participation:) }
+
+  before do
+    minor.confirm
+  end
+
+  context "when minors turn the maximum minor age" do
+    it "run gracefully" do
+      expect(Decidim::Kids::MinorAccount.where(minor:)).to be_present
+      expect(Decidim::Kids::MinorAccount.where(minor: minor_to_promote)).to be_present
+      expect { task.execute }.not_to raise_error
+    end
+
+    context "when removing minor account relation" do
+      it "removes no longer minor accounts relations" do
+        expect(organization.maximum_minor_age).to eq(13)
+        expect(minor_to_promote.minor_age).to eq(15)
```

Comment:
I think we should add 2 tests cases more:

1. when the user is exaclty one year above the age (14 years old)
2. when the user is in the last year before the upgrade (13 years old)

Reply:
Done